src/rules/converters/space-before-function-paren.ts

```diff
@@ -1,11 +1,22 @@
 import { RuleConverter } from "../converter";
 
+function isExistedESLintOption(rule: string) {
+    return ["anonymous", "named", "asyncArrow"].includes(rule);
+}
+
 export const convertSpaceBeforeFunctionParen: RuleConverter = tslintRule => {
+    const tslintRuleArguments = tslintRule.ruleArguments;
```

Comment:
There's not really much use in making a quick storage variable whose name is only one character away from its stored lookup. So you have any thoughts on removing this?

Reply:
For sure it makes sense.